tests/plugins/showblocks/showblocksonblur.js

```diff
@@ -0,0 +1,56 @@
+/* bender-tags: editor,unit */
+/* bender-ckeditor-plugins: showblocks */
+
+CKEDITOR.focusManager._.blurDelay = 0;
+
+bender.editors = {
+	editor: {
+		creator: 'inline',
+			config: {
+				showBlocksOnBlur: true
+			}
+		},
+	editorShowBlocksFalse: {
+		creator: 'inline',
+		config: {
+			showBlocksOnBlur: false
+		}
+	}
+};
+
+bender.test(
+	{
+		'test config to keep showblocks on blur': function() {
+			var bot = this.editorBots.editor,
+				editor = bot.editor;
+
+			editor.focus();
+
+			assert.isTrue(editor.editable().hasFocus);
```

Comment:
this is excessive, but it catches a case where if you run the test without the browser window being focused, editor.focus() won't have much of an effect.  (for example, because Chrome devtools takes focus for a breakpoint.  Kinda annoying.)